### ISSUE TITLE
Components do not update in server-side rendering when changes saved to source

### ISSUE THREAD

**Issue by katiefenn** [canonical-ols/javan-rhino#99]
Steps to recreate:
1) Start development site
2) Make changes to a component and save changes
3) Observe components hot-reloading
4) Refresh the page in the browser
5) Observe the "flash" between the old component look and the new component look as the app is rendered.

**Comment by bartaz** [comment]
Ad 5) There is always also a warning in browser console:

```
Warning: React attempted to reuse markup in a container but the checksum was invalid. This generally means that you are using server rendering and the markup generated on the server was not what the client was expecting. React injected new markup to compensate which works but you have lost many of the benefits of server rendering. Instead, figure out why the markup being generated is different on the client or server:
```

**Comment by earnubs** [comment]
fixed in #185
